package.json

```diff
@@ -63,6 +63,6 @@
     "@types/react": "^18.0",
     "@types/react-dom": "^18.0",
     "next-themes": "^0.2.0",
-    "typescript": "^4.9.4"
+    "typescript": "^5.0.4"
```

Comment:
Do we need to get RK back on v5 before we can merge this one in? [Upgrade wagmi, viem, typescript](https://github.com/reservoirprotocol/reservoir-kit/pull/342/files)

Reply:
This is just for local development (aka it doesn't matter what RK is on)